2-mandatory/4-stocks.js

```diff
@@ -65,6 +90,20 @@ function getPriceChanges(closingPricesForAllStocks) {
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
     // TODO
+    let arrayOfHighestStockPrice = [];
+    let highestStockPrice = [];
+    for (let i = 0; i < closingPricesForAllStocks.length; i++) {
+      highestStockPrice.push(Math.max(...closingPricesForAllStocks[i]));
+
+      arrayOfHighestStockPrice.push(
+        `The highest price of ${stocks[
+          i
```

Comment:
really well done using the available [i] here to access items in the stocks array rather than creating another loop